common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -22,14 +22,14 @@ class DB(contextWrapper: ContextWrapper) {
   private implicit val decodePublicationDate = mappedEncoding[Date, OffsetDateTime](d => OffsetDateTime.ofInstant(d.toInstant, ZoneOffset.UTC))
   private implicit val encodeStatus = mappedEncoding[RecipeStatus, String](_.toString())
   private implicit val decodeStatus = mappedEncoding[String, RecipeStatus](d => d match {
-    case "New" => New
-    case "Ready" => Ready
-    case "Pending" => Pending
-    case "Curated" => Curated
-    case "Verified" => Verified
-    case "Finalised" => Finalised
-    case "Impossible" => Impossible
-    case _ => Impossible
+    case "New" => RecipeStatusNew
```

Comment:
Rather than prefixing each one with `RecipeStatus`, you could create an object to hold them in called `RecipeStatus`. That way you would access them like `RecipeStatus.new`.